go/vt/sqlparser/ast.go

```diff
@@ -179,6 +198,20 @@ func ParseStrictDDL(sql string) (Statement, error) {
 	return tokenizer.ParseTree, nil
 }
 
+// ParseOneStrictDDL is the same as ParseOne but it errors on partially parsed
+// DDL statements.
+func ParseOneStrictDDL(sql string) (Statement, int, error) {
```

Comment:
Either now or later, let's just get rid of lenient DDL parsing, it's outlived its usefulness

Reply:
I guess just file an issue for now

Reply:
#98